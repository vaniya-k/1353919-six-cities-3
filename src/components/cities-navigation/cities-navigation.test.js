import React from 'react';
import renderer from 'react-test-renderer';
import CitiesNavigation from './cities-navigation.jsx';

it(`<CitiesNavigation/> should return 6 cities starting with Metropolis`, () => {
  const mock = jest.fn();

  const tree = renderer
    .create(<CitiesNavigation cities={[`Metropolis`, `Gotham`, `Utopia`, `Dystopia`, `Inferno`, `Limbo`]} activeCityId={0} onCityTabClick={mock}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
