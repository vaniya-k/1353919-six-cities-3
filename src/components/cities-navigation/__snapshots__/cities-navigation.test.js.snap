// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CitiesNavigation/> should return 6 cities starting with Metropolis 1`] = `
<div
  className="tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="locations__list tabs__list"
    >
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item tabs__item--active"
          href="#"
          onClick={[Function]}
        >
          <span>
            Metropolis
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item null"
          href="#"
          onClick={[Function]}
        >
          <span>
            Gotham
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item null"
          href="#"
          onClick={[Function]}
        >
          <span>
            Utopia
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item null"
          href="#"
          onClick={[Function]}
        >
          <span>
            Dystopia
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item null"
          href="#"
          onClick={[Function]}
        >
          <span>
            Inferno
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item null"
          href="#"
          onClick={[Function]}
        >
          <span>
            Limbo
          </span>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
